holidays/countries/italy.py

```
@@ -262,420 +288,667 @@
+        # Saint Bartholomew's Day.
+        self._add_holiday_aug_24(tr("San Bartolomeo apostolo"))
 
     def _populate_subdiv_bo_public_holidays(self):
-        self._add_holiday_oct_4("San Petronio")
+        # Saint Petronius's Day.
+        self._add_holiday_oct_4(tr("San Petronio"))
 
     def _populate_subdiv_br_public_holidays(self):
-        self._add_holiday_1st_sun_of_sep("San Lorenzo da Brindisi")
+        # Saint Theodore of Amasea and Saint Lawrence of Brindisi's Day.
+        self._add_holiday_1st_sun_of_sep(tr("San Teodoro d’Amasea e San Lorenzo da Brindisi"))
```

Ruff (0.14.5)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)